### PR TITLE
feat(cb2-9571):Update defect taxonomy to include m1 IVA defects

### PR DESCRIPTION
**Pull request by LGin-BJSS**
## Update defect taxonomy to include m1 IVA defects

Adds endpoint for fetching M1 IVA defect taxonomy from DynamoDB as well as associated seed data
[link to ticket number](https://dvsa.atlassian.net/browse/CB2-9571)

## Checklist

- [ ] Code has been tested manually
- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: docs/defects-api.yml [hunk 1]
```diff
@@ -4,9 +4,8 @@ info:
     This is the API spec for retrieving all defects. These defects will be
     stored locally on the VSAs mobile device. Authorization details will be
     updated once we have confirmed the security scheme we are using.
-  version: 1.0.0
+  version: 1.1.0
   title: Defect Microservice
-  termsOfService: ""
   contact:
     email: mdclarke@deloitte.co.uk
```
Comment (kcarr-bjss) [inline docs/defects-api.yml:10]: About time we changed this no?

File: output_json/M1_flattened.json [hunk 2]
```diff
@@ -0,0 +1,3373 @@
+[
```
Comment (cb-cs) [inline output_json/M1_flattened.json:1]: With iva-defects.json now in the resources directory, is this JSON still required? They look identical.

File: package.json [hunk 3]
```diff
@@ -72,6 +72,7 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
+    "@dvsa/cvs-type-definitions": "3.6.0",
```
Comment (cb-cs) [inline package.json:75]: Test result service is now using 4.0.0, are we in a position to bump this too or is it a breaking change?

File: serverless.yml [hunk 4]
```diff
@@ -50,6 +62,8 @@ custom:
         sources:
           - table: cvs-local-defects
             sources: [./tests/resources/defects.json]
+          - table: cvs-local-iva-defects
+            sources: [./output_json/M1_flattened.json]
```
Comment (cb-cs) [inline serverless.yml:66]: Similar to the question above, can this be re-pointed to resources/iva-defects.json or is there still a need for 2 separte JSON files?

File: src/config/config.yml [hunk 5]
```diff
@@ -15,6 +27,12 @@ invoke:
       getDefects:
         name: cvs-svc-defects
         mock: tests/resources/defects.json
+      getIvaDefectsByManual:
+        name: cvs-svc-iva-defects
+        mock: tests/resources/iva-defects.json
+      getIvaDefects:
+        name: cvs-svc-iva-defects
+        mock: tests/resources/iva-defects.json
```
Comment (cb-cs) [inline src/config/config.yml:35]: Just an observation, but as you can see iva-defects.json is being used here for the mock, rather than M1_flattened.json

**Pull request merged**